### ISSUE TITLE
does not work import in Python 3.6.2

### ISSUE THREAD

**Issue by ghost** [python/cpython#75578]
BPO | [31397](https://bugs.python.org/issue31397)
--- | :---
Nosy | @bitdancer
Files | <li>[1.jpg](https://bugs.python.org/file47126/1.jpg "Uploaded as image/jpeg at 2017-09-08.20:37:01 by Бобёр Сатанист")</li>

<sup>*Note: these values reflect the state of the issue at the time it was migrated and might not reflect the current state.*</sup>

<details><summary>Show more details</summary><p>

GitHub fields:
```python
assignee = None
closed_at = <Date 2017-09-09.17:17:19.223>
created_at = <Date 2017-09-08.20:37:01.134>
labels = ['type-bug']
title = 'does not work import in Python 3.6.2'
updated_at = <Date 2017-09-09.17:17:19.221>
user = None
```

bugs.python.org fields:
```python
activity = <Date 2017-09-09.17:17:19.221>
actor = 'r.david.murray'
assignee = 'none'
closed = True
closed_date = <Date 2017-09-09.17:17:19.223>
closer = 'r.david.murray'
components = []
creation = <Date 2017-09-08.20:37:01.134>
creator = '\xd0\x91\xd0\xbe\xd0\xb1\xd1\x91\xd1\x80 \xd0\xa1\xd0\xb0\xd1\x82\xd0\xb0\xd0\xbd\xd0\xb8\xd1\x81\xd1\x82'
dependencies = []
files = ['47126']
hgrepos = []
issue_num = 31397
keywords = []
message_count = 2.0
messages = ['301727', '301780']
nosy_count = 2.0
nosy_names = ['r.david.murray', '\xd0\x91\xd0\xbe\xd0\xb1\xd1\x91\xd1\x80 \xd0\xa1\xd0\xb0\xd1\x82\xd0\xb0\xd0\xbd\xd0\xb8\xd1\x81\xd1\x82']
pr_nums = []
priority = 'normal'
resolution = 'works for me'
stage = 'resolved'
status = 'closed'
superseder = None
type = 'behavior'
url = 'https://bugs.python.org/issue31397'
versions = ['Python 3.6']
```

</p></details>

**Comment by ghost** [comment]
How to import modules/files in Python 3.6.2? I started to study python, in the section about importing modules, here is an example: I created a text file with the text "print('Hello module world!')" and saved it with the name "module1.py". I put the file in a folder with a python. As stated in the book, simply entering 
"import module1" python imports the contents of the file, wherever it is on the computer, or it can be executed with the command "python module1".

In short, the method does not work on version 3.6.2,when i tried to import file, python did not find the file, neither through calling the Python interpreter from the command line, nor through IDLE. Error module not found error, and with the command 
"python  module1" - syntax error.

PATH on my computer is configured.Windows 10 ver.1703 x64,Python 3.6.2 x64

**Comment by bitdancer** [comment]
Presumably the book didn't tell you everything you need to know to make this work, or you missed something in the instructions, since this certainly does work if the module is in a directory on sys.path.  (The 'python module1.py' will only work at the windows command prompt, by the way.)

This is a bug tracker, so it's not a good place to get help on installing and using Python.  I suggest trying the pythhon-list mailing list (see mail.python.org) or the #python irc channel.
